### DISCUSSION TITLE
What are the criteria for judging air quality?

### DISCUSSION THREAD

**Discussion by plane714** [breezy-weather/breezy-weather#1529] [Q&A] [answered]
How many levels are there for air quality? What are the criteria for judging?![Screenshot_20241214-175135_Breezy Weather.png](https://github.com/user-attachments/assets/81fb305d-f476-4fe3-b18c-2511a7200060)

**Comment by papjul** [comment, marked as answer]
We follow [WHO global air quality guidelines](https://iris.who.int/handle/10665/345329).

See here for more details on how it is implemented in Breezy Weather: https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality
